doc/howto/access_ui.md

```diff
@@ -0,0 +1,39 @@
+(access-ui)=
+# How to access the LXD web UI
+
+```{important}
+Currently, the LXD web UI is included only in the `edge` channel of the LXD snap.
+
+You should never run the `edge` version of LXD in a production environment.
```

Comment:
I think "edge" here doesn't need to be quoted as it represents advice about all of the edge channels for each LXD release series.